testdata/script/tiny.txtar

```diff
@@ -21,16 +22,45 @@ garble build
 stderr '^caller: [0-9a-zA-Z_]+\.go [1-9]'
 stderr '^recovered: ya like jazz?'
 stderr 'panic: oh noes'
+stderr 'funcExported false funcUnexported false'
+stderr 'funcStructExported false funcStructUnexported false'
 -- go.mod --
 module test/main
 
 go 1.19
 -- garble_main.go --
 package main
 
-import "runtime"
+import (
+	"reflect"
+	"runtime"
+)
+
+type testStruct struct {}
+
+func (testStruct) unexportedFunc() {
+}
+
+func (testStruct) ExportedFunc() {
+}
+
+func ExportedFunc() {
+}
+
+func unexportedFunc() {
+}
```

Comment:
perhaps make these funcs have a dummy body, like a `println`, to ensure they're not discarded as no-ops.

they can each occupy a single line, too, to avoid verbosity.